<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1754</width>
    <height>997</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="vertexsize_hi">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Hi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertexcolor_red">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>90</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>--VertexType--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertexcolor_green">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>90</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edgecolor_green">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>310</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QPushButton" name="perspective">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>750</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Perspective</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>--VertexColor--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edgecolor_blue">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>310</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stipple_off">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>790</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stipple OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertexsize_lo">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Lo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertextype_square">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>240</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
   </widget>
   <widget class="QPushButton" name="background_grey">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>450</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Grey</string>
    </property>
   </widget>
   <widget class="QPushButton" name="background_white">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>450</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>White</string>
    </property>
   </widget>
   <widget class="QPushButton" name="crazy_color">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>710</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Crazy Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="standart_color">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>710</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Standart Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stipple_on">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>790</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stipple ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertexcolor_blue">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>90</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="background_black">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>450</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>520</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>--Rotation--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertexsize_mid">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Mid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edgecolor_hi">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>380</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Hi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>--EdgeColor--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertextype_round">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>240</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Round</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-Scale-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_25">
    <property name="geometry">
     <rect>
      <x>1651</x>
      <y>1041</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Screenshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edgecol_red">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>310</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orthographic">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>750</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Orthographic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>--VertexSize--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>350</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>--EdgeSize--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>420</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>--Background--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edgesize_lo">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>380</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Lo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edgesize_mid">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>380</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Mid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_obj">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>0</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Open .obj</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>550</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>550</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>550</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>600</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>--Moving--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>630</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>630</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>630</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="scale">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="rot_x">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>570</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="rot_y">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>570</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="rot_z">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>570</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="move_z">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>650</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="move_x">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>650</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="move_y">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>650</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="change_move">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>640</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_rot">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>560</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>761</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1754</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
